test-data/unit/semanal-types.test

```diff
@@ -736,7 +736,7 @@ MypyFile:1(
       f
       Args(
         Var(a))
-      def (a: builtins.Any) -> builtins.Any
+      def (a: Any) -> Any
```

Comment:
This change is new. I'm not comfortable with both the original and the new version.